utils/api/definitions/schemas.ts

```diff
@@ -214,13 +212,13 @@ export const RequestSchema = Joi.object({
     .custom((value, helpers) => {
       const { livingCountry, yearsInCanadaSince18 } = helpers.state.ancestors[0]
       if (livingCountry === 'CAN' && yearsInCanadaSince18 !== undefined) {
-        if (yearsInCanadaSince18 < 10) {
-          return helpers.message({
-            custom: value
-              ? ValidationErrors.socialCountryUnavailable10
-              : ValidationErrors.yearsInCanadaNotEnough10,
-          })
-        }
+        // if (yearsInCanadaSince18 < 10) {
+        //   return helpers.message({
+        //     custom: value
+        //       ? ValidationErrors.socialCountryUnavailable10
+        //       : ValidationErrors.yearsInCanadaNotEnough10,
+        //   })
```

Comment:
you don't get an error for an empty if ??

Reply:
no error. Will probably delete though